apps/test/unit/lib/kits/maker/commandsTest.js

```diff
@@ -4,19 +4,20 @@ import sinon from 'sinon';
 import {
   analogRead,
   analogWrite,
+  boardConnected,
   digitalRead,
   digitalWrite,
   injectBoardController,
   onBoardEvent,
   pinMode
 } from '@cdo/apps/lib/kits/maker/commands';
-import CircuitPlaygroundBoard from '@cdo/apps/lib/kits/maker/CircuitPlaygroundBoard';
+import FakeBoard from '@cdo/apps/lib/kits/maker/FakeBoard';
```

Comment:
Both `CircuitPlaygroundBoard` and `FakeBoard` are required to conform to the new `MakerBoard` interface as of my last PR, and we're already stubbing the whole class, below - so I've swapped this test to depend on `FakeBoard` which means fewer test dependencies and is equally valid for unit testing purposes.